packages/o365/data_stream/audit/elasticsearch/ingest_pipeline/default.yml

```
@@ -1770,6 +1796,75 @@ processors:
       copy_from: o365audit.ApplicationDisplayName
       tag: set_application_name
       ignore_empty_value: true
+
+  # ExchangeItemAggregated Schema
+  - append:
+      field: event.type
+      value: access
+      if: ctx.o365audit?.RecordType != null && ctx.o365audit?.RecordType == "50"
```

```suggestion
      if: ctx.o365audit?.RecordType == "50"
```
If `ctx.o365audit?.RecordType` is "50" it cannot be `null`, so the first clause is redundant.